aes-gcm/src/lib.rs

```diff
@@ -108,11 +108,17 @@
 //! [`aead::Buffer`] for `arrayvec::ArrayVec` (re-exported from the [`aead`] crate as
 //! [`aead::arrayvec::ArrayVec`]).
 
+#[cfg(feature = "aes")]
+mod xaes;
+
 pub use aead::{self, AeadCore, AeadInPlace, Error, Key, KeyInit, KeySizeUser};
 
 #[cfg(feature = "aes")]
 pub use aes;
 
+#[cfg(feature = "aes")]
```

Comment:
Any reason it is under feature = "aes" ? Any reason it can't be separate feature = "xaes" extending "aes" ?

Reply:
Whatever is desired!